content/en/hosting/kubernetes-vs-docker.md

```diff
@@ -121,3 +109,9 @@ This deployment hosts 46 production instances on Amazon [Elastic Kubernetes Serv
 To give more specific examples of what's running in the 46 instances, here's details on two including their [Apdex](https://docs.communityhealthtoolkit.org/building/guides/database/querying_apdex_telemetry/):
 * One large instance has 2.4k active users, 8.9M documents in the Medic database in CouchDB and a mean replication Apdex of 80.0.
 * Another medium instance has 232 active users, 279k documents in the Medic database in CouchDB and a mean replication Apdex of 56.4.
+
+## Which one to choose?
+
+[Docker](/hosting/4.x/docker/) is suited for most CHT deployments use cases. Use [Kubernetes](/hosting/4.x/kubernetes/) for multi-tenant or specific hardware constrained deployments, such as bare-metal servers with a low core-count.
+
+For more details on the research behind why Docker and single-node CouchDB are preferrable for CHT deployment, see [this forum post](https://forum.communityhealthtoolkit.org/t/investigate-adding-more-shards-as-a-potential-avenue-for-improved-performance/4831?u=mrjones).
```

Comment:
I love the simplicity of this section! Can you move it to the top of the page?  I feel like we're [burying the lede](https://www.dictionary.com/e/slang/bury-the-lede/) as is.

Reply:
Good call! Handled in https://github.com/medic/cht-docs/pull/1891/commits/7d774c18943a85306e4cdf46dad4b00412e8dca3